src/overview/assets/index.html

```diff
@@ -15,20 +15,97 @@ <h1>Java Overview</h1>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="jdkVersion" id="openjdk12Radio" value="openjdk12">
+                        <label class="form-check-label" for="openjdk12Radio">
+                          OpenJDK 12 (Latest)
+                        </label>
+                      </div>
+                    </div>
+                    <div class="form-group col-sm-6">
+                      <label for="inputEmail4">JVM:</label>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="jvmImpl" id="hotspotRadio" value="hotspot" checked>
+                        <label class="form-check-label" for="hotspotRadio">
+                          Hotspot
+                        </label>
+                      </div>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="jvmImpl" id="openj9Radio" value="openj9">
+                        <label class="form-check-label" for="openj9Radio">
+                          OpenJ9
+                        </label>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="form-row">
+                    <div class="form-group col-sm-12">
+                      <div class="spinner-border spinner-border-sm" role="status" id="jdkSpinner">
+                        <span class="sr-only">Loading...</span>
+                      </div>
+                      <a class="btn btn-primary btn-lg d-none" id="jdkDownloadLink" href="command:java.helper.openUrl" role="button" title="Download JDK">
+                          Download
+                          <br>
+                          <sub id="jdkReleaseName">release</sub><sub> | </sub><sub id="jdkDownloadSize">size</sub>
+                      </a>
+                    </div>
+                  </div>
+                </form>
+              </div>
+              <div class="tab-pane fade" id="otherJdkPanel" role="tabpanel" aria-labelledby="otherJdkTab">
+                <ul id="jdkProviderList" class="list-unstyled">
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.azul.com%2Fdownloads%2Fazure-only%2Fzulu%2F%22" title="Recommended for Microsoft Azure Cloud and Azure Stack applications">Azul Zulu Enterprise for Azure</a></li>
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.oracle.com%2Ftechnetwork%2Fjava%2Fjavase%2Fdownloads%2Findex.html%22" title="Java SE Downloads by Oracle">Java SE Downloads by Oracle</a></li>
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fopenjdk.java.net%2Finstall%2F%22" title="OpenJDK">OpenJDK</a></li>
+                </ul>
+              </div>
+            </div>
+            <p>When you are finished, please reload Visual Studio Code to activate Java language server.</p>
```

Comment:
`Java language server` or `Language Support for Java`?

I prefer the latter one since that's the official name of the extension.